astropy/io/votable/validator/result.py

```diff
@@ -14,7 +14,8 @@
 import subprocess
 import warnings
 import pickle
-import urllib
+import urllib.request
+import urllib.parse
```

Comment:
it's urllib.error

Reply:
Sorry for the squash but this should be resolved now.

Reply:
but urllib.parse isn't used, or is it in a tricky way?

Reply:
Gah, I'm sorry.